Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [package]
-name = "structured_headers"
+name = "sfv"
+description = "Structured Field Values for HTTP. Implementation of IETF draft https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html v19"
```

Comment:
Maybe remove the v19 from the description? I don't know if that's relevant, and it'll be easy to miss when updating.

Reply:
Okay, I'll do. I put it to indicate that the implementation covers the requirements of the 19th version. Cause maybe there will be more versions...)